dali/pipeline/util/bounding_box.h

```diff
@@ -153,6 +156,25 @@ class BoundingBox {
     return {left_, top_, right_ - left_, bottom_ - top_};
   }
 
+  void CopyAsCenterWh(float *destination) const {
```

Comment:
What is this function exactly doing?

Reply:
Fixed by creating `AsCenterWh` method handling format change and `WriteBoxToOutput` method of `BoxEncoder` handling actual copying to output.